packages/dds/sequence/src/SharedStringWithSequentialIdFactory.ts

```diff
@@ -0,0 +1,55 @@
+import * as MergeTree from "@fluidframework/merge-tree";
+import { IChannelAttributes, IChannelServices,
+    IFluidDataStoreRuntime, IChannelFactory } from "@fluidframework/datastore-definitions";
+import { pkgVersion } from "./packageVersion";
+import { sharedStringWithSequentialIdMixin } from "./sharedStringWithSequentialId";
+import { SharedString } from "./sharedString";
+
+export class SharedStringWithSequentialIdFactory implements IChannelFactory {
+    // TODO rename back to https://graph.microsoft.com/types/mergeTree/string once paparazzi is able to dynamically
```

Comment:
Regarding this todo... copied from SharedStringFactory... I assume it is still needed here?